sdk/servicebus/Azure.Messaging.ServiceBus/src/Client/ServiceBusClient.cs

```diff
@@ -37,7 +40,11 @@ public class ServiceBusClient : IAsyncDisposable
         /// <value>
         ///   <c>true</c> if the client is closed; otherwise, <c>false</c>.
         /// </value>
-        public bool IsClosed { get; private set; }
+        public virtual bool IsClosed
```

Comment:
Changed semantics here to improve thread safety.   @JoshLove-msft:  Can you confirm there was no specific reason why this wasn't `volatile` ?

Reply:
Nope, was likely an oversight.